L/libgraphicsmagic/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libgraphicsmagic"
+version = v"1.3.36"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("ftp://ftp.icm.edu.pl/pub/unix/graphics/GraphicsMagick/1.3/GraphicsMagick-1.3.36.tar.gz", "1e6723c48c4abbb31197fadf8396b2d579d97e197123edc70a4f057f0533d563")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/GraphicsMagick*/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --enable-shared
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(!Sys.iswindows, supported_platforms(; experimental=true))
```

Comment:
Do you want to see whether Windows just works?  Many eople don't consider cross-compilation to be a thing and reinvent a different build-system just for Windows.  Autoconf is usually very robust for cross-compilation, if set up properly.  If it doesn't work we get back here.

Reply:
it errored, looking at: http://www.graphicsmagick.org/INSTALL-windows.html#installing-from-source-code
I think there are some work to be done but I'm not familiar with Windows/Visual C++ stack so I think I will skip for now. I'm adding this package for `timg` and `timg` is only really useful on Linux

Reply:
Ok, fair enough 👍